src/applications/financial-status-report/tests/unit/transform.unit.spec.jsx

```diff
@@ -0,0 +1,658 @@
+            letterCode: '100',
+            description:
+              'First Demand Letter - Inactive Benefits - Due Process',
+          },
+        ],
+        id: 2,
+        resolution: {
+          resolutionType: 'Waiver',
+          agreeToWaiver: true,
+        },
+      },
+    ],
+    debt: {
+      currentAr: 0,
+      debtHistory: [
+        {
+          date: '',
+        },
+      ],
+      deductionCode: '',
+      originalAr: 0,
+    },
+    currEmployment: [
+      {
+        veteranGrossSalary: '5000.54',
+        deductions: [
+          {
+            name: 'Federal tax',
+            amount: '350.56',
+          },
+          {
+            name: 'State tax',
+            amount: '230.45',
+          },
+          {
+            name: 'Pension',
+            amount: '100',
+          },
+          {
+            name: 'Health insurance',
+            amount: '254.45',
+          },
+        ],
+        type: 'Full time',
+        from: '2017-1-XX',
+        to: '',
+        isCurrent: true,
+        employerName: 'Veteran Current One',
+      },
+      {
+        veteranGrossSalary: '2000.56',
+        deductions: [
+          {
+            name: 'Federal tax',
+            amount: '200.02',
+          },
+          {
+            name: 'Allotment',
+            amount: '134.56',
+          },
+          {
+            name: 'FICA (Social Security and Medicare)',
+            amount: '122.4',
+          },
+        ],
+        type: 'Part time',
+        from: '2019-7-XX',
+        to: '',
+        isCurrent: true,
+        employerName: 'Veteran Current Two',
+      },
+    ],
+    spCurrEmployment: [
+      {
+        spouseGrossSalary: '4000.45',
+        deductions: [
+          {
+            name: 'Federal tax',
+            amount: '325.54',
+          },
+          {
+            name: 'Health insurance',
+            amount: '250.34',
+          },
+          {
+            name: 'Garnishment',
+            amount: '150',
+          },
+        ],
+        type: 'Full time',
+        from: '2018-6-XX',
+        to: '',
+        isCurrent: true,
+        employerName: 'Spouse Current',
+      },
+    ],
+    financialOverview: {},
+    availableOptions: {},
+    income: [
+      {
+        veteranOrSpouse: 'VETERAN',
+        compensationAndPension: '3261.1',
+        education: '50',
+      },
+    ],
+    AGREED: true,
+  },
+};
+describe('fsr transform helper functions', () => {
+  describe('sumValue helper', () => {
+    it('should sum specified object keys in list of objects', () => {
+      const objectList = Array(5).fill({ someKey: '10' });
+      expect(sumValues(objectList, 'someKey')).to.equal(50);
+    });
+  });
+  describe('dateFormatter helper', () => {
+    it('should format the date', () => {
+      expect(dateFormatter('2004-10-XX')).to.equal('10/2004');
+    });
+  });
+});
+
+describe('fsr transform information', () => {
+  const submissionObj = JSON.parse(transform(null, someInputObject));
```

Comment:
I have mixed feelings about this. I see why its here, but really, the test should run the full logic itself and be completely isolated.

Each test should follow the [AAA pattern](https://blog.logrocket.com/javascript-testing-best-practices/#:~:text=Structure%20tests%20with%20the%20AAA%20pattern)